res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+		<programmerole guid="c2edde3f-c2ff-4800-a020-79f59662176f" first_name="George" last_name="Washington" gender="1" />
+		<programmerole guid="29aa9b04-b155-4975-b1b0-45355c293668" first_name="John" last_name="Hancock" gender="1" />
+		<programmerole guid="c4f29c58-9028-48d4-84de-cad66012d2e4" first_name="Thomas" last_name="Jefferson" gender="1" />
+
+		<programmerole guid="358e9bb3-32b9-4df7-aaa4-b47271c58180" first_name="Zed Mack" last_name="Zedrick" gender="1" />
+		<programmerole guid="e9e3fc6b-f3c9-48be-852c-8b079f1b8e88" first_name="Amy" last_name="Harris" gender="2" />
+		<programmerole guid="833302a7-87ee-4afc-95db-f0fde5e7866a" first_name="Lilly" last_name="Berner" gender="2" />
+		<programmerole guid="7ab5fea3-0c9a-4973-bf04-c7c2cf30382e" first_name="Madison" last_name="Japens" gender="2" />
+		<programmerole guid="cf8c6448-47ec-4a8b-941a-5a390ff76f04" first_name="Elvis" last_name="" gender="1" />
+
+		<programmerole guid="f1fa679f-cb36-4d36-bdd7-c105b9605be4" first_name="Dink" last_name="" gender="1" />
+		<programmerole guid="ac69a50f-22e0-4384-a2e4-24485b66a1ad" first_name="Mimikry" last_name="" gender="2" />
+		<programmerole guid="d2168e7c-3c95-4778-b406-cb305633946a" first_name="Trint" last_name="" gender="1" />
+		<programmerole guid="fbff6c7a-6795-4e2f-9a55-dd9b833945d1" first_name="Dar" last_name="" gender="1" />
+		<programmerole guid="53104872-6c35-43df-a8b6-a3863db9cf9b" first_name="Isolde" last_name="" gender="2" />
+		<programmerole guid="8a54557b-ded8-4427-bb60-7796c1f84317" first_name="Arturo" last_name="" gender="1" />
+
+		<programmerole guid="6e1f331a-7c9d-433b-bc9f-7606ad322a5e" first_name="Raymond" last_name="Plankwood" gender="1" />
+		<programmerole guid="3ee1d44a-f3fc-47a5-aabf-25803e6e9f99" first_name="Evelyn" last_name="Harley" gender="2" />
+		<programmerole guid="410a932c-9ceb-4080-9dbc-f82ac3c9bcd3" first_name="P.G." last_name="LeCruck" gender="1" />
+		
+		<programmerole guid="be9ab541-b49d-4563-a869-a9cdc90c63f8" first_name="Stevan" last_name="" gender="1" />
+		<programmerole guid="a42f6980-1685-422c-8796-8b502b9b41d9" first_name="Kara" last_name="" gender="2" />
+		<programmerole guid="d783dea1-1f41-496f-a239-12ac44c6fe60" first_name="Cellist" last_name="" gender="1" />
+
+		<programmerole guid="48d44fca-7d78-4f47-b22b-7aae5364f12b" first_name="Leather Jacket Leo" last_name="" gender="1" />
+		<programmerole guid="02e4e684-131f-4e7a-b8ef-de492a2e9b50" first_name="Fervent Faviola" last_name="" gender="2" />
+		
+		<programmerole guid="437539e5-4c6b-4207-981a-051a82021745" first_name="Bobby" last_name="Blaze" gender="1" />
+		<programmerole guid="ba05561a-0266-4cfe-b020-353781fd881c" first_name="Xorp" last_name="" gender="0" />
+	</programmeroles>
+
+	<scripttemplates>
+		<scripttemplate product="3" licence_type="1" guid="e569b0af-e97f-4897-8b8d-6c426fa5a7d5" creator="9551" comment="Low-Budget- und Endlos-Variante von scripttemplate-ron-morningshow">
```

Comment:
You would use it if you need a morning show but the other isn't available (eg if we set things to be available only once simultaneously)

Reply:
But why would you "need" a morning show? You can always broadcast something else that can pay for itself.
I want to prevent putting work into a template that is probably doomed (never used).

Reply:
Maybe it allows certain ads ("Der Wachmacherkaffee"  - broadcast only between 6-10, target group seniors, ...).

Not needed "now" but the basic approach ("cheap production stuff" vs "expensive ones") is not unimportant - not every script is targeting a blockbuster production.

Yet I understand your point on avoiding unnecessary things.

Reply:
That program is more for the lulz, but it *is* very cheap to produce (cheapest version can be below $30,000, and it's 3 blocks — although I saw discussion that this might be a bug). Still, broadcasting a programme that you can sell is probably almost always more rational in the early phase, but a game can also have some red herrings.

Reply:
We will need to check this again once the big changes regarding custom productions are done. One complaint was that the production cost does not increase with the number of costs - when implemented productio may be much more expensive in this case.